tests/tox_tests/test_med_00_client.py

```diff
@@ -6,7 +6,7 @@
 
 
 @patch('tempfile.TemporaryDirectory.__enter__', return_value=get_sample_file_path(''))
-def test_get_file_handle(patch_temp, sftp_client):
+def test_get_file_handle(sftp_client):
```

Comment:
@radbrt I think that `patch_temp` parameter is required even though its not necessarily referenced in this test since the [patch decorator](https://realpython.com/python-mock-library/#patch) returns a mock object. If we remove it, it looks like the test still passes but were accidentally assigning the mock object to the sftp_client so its not actually testing what we wanted to anymore and I think always passes.

I raised a few exceptions locally to confirm my expectations. Without the patch_temp arg the stfp_client doesnt run the actual code, since its a MagicMock object:

![Screen Shot 2022-09-19 at 9 34 06 AM](https://user-images.githubusercontent.com/27376735/191029589-1ed62470-f144-4e26-9316-b5409910257b.png)

With the patch_temp arg we see that stfp_client is the real SFTPConnection we want to use to test.

![Screen Shot 2022-09-19 at 9 34 12 AM](https://user-images.githubusercontent.com/27376735/191029591-cc14abd2-41ad-4fd3-87ef-05cc9eb91e84.png)

We can either add a `patch_temp.assert_called()` to assert that a temp file was attempted to be created or we can use something like and underscore to represent the positional argument while showing that its not actually used like `def test_get_file_handle(_, sftp_client):`, like a placeholder.

Does that make sense? Am I understanding correctly?

Reply:
@pnadolny13 that makes a lot of sense really, I too thought the argument was needed as a placeholder, but when it still passed after I tried to remove it, I assumed there was something about mocks I hadn't understood. I guess that's kind of true 🤔  Let's keep it, I'll push an update.